integr8ly/integreatly-operator
# MGDAPI-3794: Remove G101 from makefile exclude list

# Issue link
https://issues.redhat.com/browse/MGDAPI-3794

# What
Remove any occurences of hard coded credentials, and remove G101 from the gosec exclude list in the makefile.

# Verification steps
Verify that the gosec prow check passes on this PR, and that all lines of code marked as false positives are in fact false positives (ie they do not contain hard coded credentials).
